spec/lib/whitehall_importer/embed_body_references_spec.rb

```diff
@@ -30,6 +30,16 @@
       expect(govspeak_body).to eq("Bar  Baz")
     end
 
+    it "ensures that one new line before an image is replaced by two" do
+      govspeak_body = described_class.call(
+        body: "hello\r\n!!1\n!!2",
+        images: ["foo.png", "bar.jpg"],
+      )
+
+      expect(govspeak_body)
+        .to eq("hello\r\n\n[Image:foo.png]\n\n[Image:bar.jpg]")
```

Comment:
This comes out a little whacky with the mix of windows and unix line breaks in the first example where I'd expect `\r\n\r\n`

Reply:
Yes, it is a bit funky!  I was trying to show that `\r`'s aren't touched by this code, but may exist.  However, what I didn't want to do is write duplicate tests (with and without `\r`).  Is the best (most pragmatic) solution to simply remove all `\r`'s first?